frame/system/src/tests.rs

```diff
@@ -436,21 +436,6 @@ fn runtime_upgraded_with_set_storage() {
 	});
 }
 
-#[test]
-fn events_not_emitted_during_genesis() {
```

Comment:
any tricks to allow this test to exist without creating a new one-off feature flag?